libafl/src/executors/command.rs

```diff
@@ -84,10 +84,11 @@ pub enum InputLocation {
 /// Writes the input either to stdio or to a file
 /// Use [`CommandExecutor::builder()`] to use this configurator.
 #[derive(Debug)]
-pub struct StdCommandConfigurator {
+pub struct StdCommandConfigurator<TC> {
```

Comment:
BC (BytesConverter)?

Reply:
yeah but i won't continue on this. i'll put this instate.